docs/backend/tracking.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Backend Tracking
```

Comment:
I removed the `Backend` from this so that the sidebar doesn't repeat itself with a `Backend Tracking` page within the `Backend` section